components/canvas/NodeDescription.tsx

```diff
@@ -19,7 +19,12 @@ export const NodeDescription = ({ header, description }: NodeDescription) => {
       )}
       {description && (
         <Typography variant="caption" className={styles["node__description"]}>
-          {formatNodeText(description)}
+          <FormatNodeText
+            variant="caption"
+            className={styles["node__description"]}
+          >
+            {description}
+          </FormatNodeText>
         </Typography>
```

Comment:
The Typography wrapper can probably be deleted now

Reply:
deleted